src/cmd/package.go

```diff
@@ -143,6 +144,35 @@ func newPackageCreateCommand(v *viper.Viper) *cobra.Command {
 	return cmd
 }
 
+// Converts registry overrides to a structured type.
+// The result will be sorted in descending order.
+// Descending order guarantees the longest prefix will be sorted toward the beginning.
+//
+// Input is of the following form:
+// []string{"docker.io/library=docker.example.com", "docker.io=docker.example.com"}
+func parseRegistryOverrides(overrides []string) ([]types.RegistryOverride, error) {
```

Comment:
The only other item that stands out to me is guarantees on precedence. Do we know how we intend to handle duplicate sources (should they exist)?

Additionally considering if we should update the language for the command (currently states it is a `map`)